spec/controllers/questions/consent_controller_spec.rb

```diff
@@ -202,6 +202,31 @@
         expect(error_messages[:primary_last_name].first).to eq "Please enter your last name."
       end
     end
+
```

Comment:
I'd like to see a test covering the fact that `ConsentController` now is responsible for adding `primary_consented_to_service_at`.

Reply:
i expected that it already did, but it only tested the ip_address getting saved. I'll add that.